internal/pkg/errors/error_message.go

```diff
@@ -61,6 +61,7 @@ const (
 	InvalidCloudErrorMsg               = "error defining plugin on given Kafka cluster"
 	InvalidCloudSuggestions            = "To list available connector plugin types, use `confluent connect plugin list`."
 	ConnectLogEventsNotEnabledErrorMsg = "Connect Log Events are not enabled for this organization"
+	PluginInvalidExtensionErrorMsg     = "Only ZIP/JAR plugin file is allowed"
```

Comment:
```suggestion
	PluginInvalidExtensionErrorMsg     = "only ZIP/JAR plugin file is allowed"
```
We only capitalize proper nouns in our error messages.